### PR TITLE
Closure function column

### PR DESCRIPTION
**Pull request by dannoso**
So simple so useful.

### CODE REVIEW

File: src/resources/views/columns/closure_function.blade.php [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{{-- closure function --}}
+<span>
```
Comment (pavoltanuska) [inline src/resources/views/columns/closure_function.blade.php]: Why not <td>?

**Comment by pavoltanuska** [comment]
What is the advantage of this over doing it with `model_function` column?

**Comment by dannoso** [comment]
@pavoltanuska Honestly I don't think those functions belong to model file. I generally don't like functions producing html in models. On top of this sometime you need that column format only in one crudController. Having it in the model is pointless. I felt was a missing feature might be interesting for other users too.

File: src/resources/views/columns/closure_function.blade.php (hunk 1 above)
Comment (dannoso) [reply src/resources/views/columns/closure_function.blade.php]: <td> is already added by datatable. Many other columns don't have <td></td> i.e. model_function

Comment (pavoltanuska) [reply src/resources/views/columns/closure_function.blade.php]: You're right, my bad, the `<td>` to `<span>` rewrite was introduced with datatables-responsive plugin.

**Comment by pavoltanuska** [comment]
On second thought, I think you're right and this can come handy. Seems way cleaner, thanks for your reply :)

File: src/resources/views/columns/closure_function.blade.php [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+	{!! $column['function']($entry) !!}
```
Comment (pavoltanuska) [inline src/resources/views/columns/closure_function.blade.php]: Indent with spaces, to stay consistent with other column types (even though indentation seems all over the place in some other fields).

**Comment by tabacitu** [comment]
I love this. You're totally right @dannoso - this should prove useful. Thank you for reviewing it @pavoltanuska . 

I've just tested it and renamed it from ```closure_function``` to ```closure``` - seemed simpler to me, since a closure _is_ a function, after all. Please let me know if you think ```closure_function``` was better and why - I haven't tagged it yet so we still have time to make changes.

For reference, here's a valid declaration:

```php
$this->crud->addColumn([
    'name' => 'created_at',
    'label' => 'Created At',
    'type' => 'closure',
    'function' => function($entry) {
        return 'Created on '.$entry->created_at;
    }
]);
```

Thanks a lot. Cheers!

**Pull request merged**